ipaplatform/ubuntu

```diff
@@ -0,0 +1 @@
+debian
```

Comment:
Is this a symlink? GH UI doesn't clearly indicate the type.

It should not be necessary to add a symlink here. IPA tries ``ID_LIKE`` and Ubuntu should define ``ID_LIKE=debian`` in ``/etc/os-release``.

Reply:
it is, and the failure came from pylint IIRC, build etc all would work fine without this

though it could be probably fixed in some other way too?

Reply:
I've dropped it for now, need to check where it actually failed and then send separately to see if it can be fixed in some other way

Reply:
It still shows up.